GACManager/GACManager/GACManager/GACManagerView.xaml.cs

```diff
@@ -55,57 +58,81 @@ void HelpCommand_Executed(object sender, Apex.MVVM.CommandEventArgs args)
 
         void UninstallAssemblyCommand_Executing(object sender, Apex.MVVM.CancelCommandEventArgs args)
         {
-            //  Double check with the user.
-            args.Cancel = MessageBox.Show("Are you sure you want to uninstall this assembly?", "Are you sure?",
-                                          MessageBoxButton.YesNoCancel)
-                          != MessageBoxResult.Yes;
+            if (MessageBox.Show("Are you sure uninstall selected assemblies?", "Are you sure?", MessageBoxButton.YesNo)
+                != MessageBoxResult.Yes)
+            {
+                args.Cancel = true;
+                return;
+            }
+
+            args.Parameter = listView.SelectedItems;
         }
 
         void UninstallAssemblyCommand_Executed(object sender, Apex.MVVM.CommandEventArgs args)
         {
-            //  The parameter must be an assembly.
-            var assemblyViewModel = (GACAssemblyViewModel)args.Parameter;
+            var sb = new StringBuilder();
+            var okCount = 0;
+            var errCount = 0;
+
+            //todo progress report
+            foreach (var model in (args.Parameter as IList)!.Cast<GACAssemblyViewModel>())
+            {
+                if (ProcessUninstall(model, out var errorMsg))
+                {
+                    //  Remove the assembly from the vm.
+                    ViewModel.Assemblies.Remove(model); //todo not work

```

Comment:
Does this not work as well?

Reply:
Yes, it's not work on my environment, Windows Server 2008 R2. it's a little issue, not important, I just record it, you know everyone is busy, I understand. Have a nice day.